src/main/java/PasswordGenerator.java

```diff
@@ -22,18 +21,24 @@ public String createStringOfRandomCharacters() {
 
         String randomCharacters = "";
 
+        //This could be a for loop
         randomCharacters += pickCharactersFromList(1, characterTypeList.get(0));
         randomCharacters += pickCharactersFromList(1, characterTypeList.get(1));
         randomCharacters += pickCharactersFromList(1, characterTypeList.get(2));
 
         return randomCharacters;
     }
 
+    //method visibility?
     public String pickCharactersFromList(int numberAmount, String alphabet) {
+        //variable naming?
         int N = alphabet.length();
+        //maybe character isn't the best name?
         String character = "";
```

Comment:
If I'm reading it right _character_ actually holds multiple characters so maybe the plural would make more sense.
Seems petty but helps with the understanding.